### ISSUE TITLE
Incorrect html output from IBM i

### ISSUE THREAD

**Issue by Dizzy-cz** [pmattes/x3270#92]
Hello,

I have problem with exporting some "fancy" screens to HTML, seems to be problem with exporting "inverse" characters.

On x3270/wc3270 is everything displayed correctly (screenshot x3270.png), but HTML file is created incorrectly, whole deep sky blue "frame" is inverted (html_export.png). When I change color to background color and vice versa (for certain parts in html file) everything looks fine.

I don't know if this is bug or incorrect settings, but I can't find any relevant settings.
I know support of IBM i (AS/400) isn't main goal for this project, but will be nice if these screens are exported correctly. :)

Correct x3270 rendening
![x3270](https://github.com/user-attachments/assets/05c411cd-6ee2-430e-8689-8a7adcbe137c)

Incorrect HTML export.
![html](https://github.com/user-attachments/assets/49a7fcf1-3a99-4411-9be3-bb4215c4c666)

Incorrect HTML export with element inspector
![html_export](https://github.com/user-attachments/assets/91aa3c2f-c8d9-4c6f-b335-32f58b38e88c)

**Comment by pmattes** [comment]
The HTML export is not protocol-specific -- it's supposed to work properly for 3270, 5250 (which x3270 isn't explicitly aware of anyway), and NVT. So something's definitely wrong.

Is there any chance you can capture a trace when this happens? I just need you to turn on tracing when the screen has been drawn by the host, and then turn it off. The trace will contain the screen contents. Perhaps you could send it to me directly instead of posting it on GitHub, if there are confidentiality issues.

Without the trace info, I'm just staring at the HTML rendering code and guessing, so it could take quite a while to figure out.

**Comment by Dizzy-cz** [comment]
Hello, 
I send you e-mail with some info, but I'm not sure if this is the correct way. I use your email address on university.

Best regards

**Comment by pmattes** [comment]
No, I don't see anything. Did you send it to paul.mattes@case.edu?
